java/src/org/openqa/selenium/chrome/ChromeDriverFinder.java

```diff
@@ -0,0 +1,25 @@
+package org.openqa.selenium.chrome;
+
+import org.openqa.selenium.remote.service.DriverFinder;
+
+import java.io.File;
+
+import static org.openqa.selenium.chrome.ChromeDriverService.CHROME_DRIVER_EXE_PROPERTY;
+public class ChromeDriverFinder extends DriverFinder {
+  protected static File findExecutable() {
+    return findExecutable(
+      "chromedriver",
+      CHROME_DRIVER_EXE_PROPERTY,
+      "https://chromedriver.chromium.org/",
+      "https://chromedriver.chromium.org/downloads");
+  }
+
+  protected static File findExecutable(ChromeOptions options) {
+    return findExecutable(
+      "chromedriver",
+      CHROME_DRIVER_EXE_PROPERTY,
+      "https://chromedriver.chromium.org/",
+      "https://chromedriver.chromium.org/downloads",
+      options);
+  }
+}
```

Comment:
I am not following why we need this class. Why don't we call `DriverFinder.findExecutable` in `ChromeDriver`? We could put all hard coded values in variables, make the call from there and avoid this extra class. 

This comment also applies for the rest of the drivers.